src/backend/distributed/utils/maintenanced.c

```diff
@@ -731,6 +736,7 @@ CitusMaintenanceDaemonMain(Datum main_arg)
 				foundDeadlock = CheckForDistributedDeadlocks();
 			}
 
+
```

Comment:
nit: can you undo this empty line addition?

Reply:
Done